### PR TITLE
DOC-107 Assets Refresh

### PR DESCRIPTION
**Pull request by janettelynch**
@karsonkalt, @sarabeth, please review my updates and let me know if I omitted features or other changes. Thanks!

### CODE REVIEW

**Review by karsonkalt — COMMENTED** [review]
Hey Janette, verbiage looks great! I just have a couple of recs wit the screenshots:
- [x] The new `/asset-property-filter-select.png` is a screenshot of the entity drawer. The new Property drilldown is not in the header and is under Class and Type
- [x] I pushed up a change yesterday that shows the Asset Group at the top of the filer drawer for all users. I think you should be able to see that and get it in the `/asset-quick-filter-type.png` screenshot

Thanks so much!

File: knowledgeBase/asset-management/asset-inventory-filters.md [hunk 1]
```diff
@@ -1,50 +1,70 @@
...
+
 
 When you select any of the assets in the filtered results table, its detailed properties side panel opens on the right.
 
 ![](../assets/asset-selected-entity-properties.png)
 
 ## Granular Filtering by Properties
 
-After you have filtered on class and type, you can apply granular filters by selecting specific property values.
+After you have filtered on class, type, and property, you can apply granular filters to a specific asset you have selected in the list of results.
```
Comment (karsonkalt) [inline knowledgeBase/asset-management/asset-inventory-filters.md:62]: Should we capitalize Class, Type, and Property to be consistent throughout the docs?

File: knowledgeBase/asset-management/asset-inventory-filters.md [hunk 2]
```diff
@@ -1,50 +1,70 @@
...
+1. Click the three-dot icon  ![](../assets/icons/assets-more.png)  in the top-right corner.
+2. Select the filters you want to apply.
 
-1. Click **Filter** ![filter](../assets/icons/filter.png) in the top-right corner.
-2. Find all the properties you want to filter on.
-3. Select the required value for each property.
-4. Click **Filter** ![green-filter](../assets/icons/green-filter.png) at the bottom.
-   ![](../assets/asset-property-filter-select.png)
+   ​
+   ![](../assets/asset-property-filter-select.png)
```
Comment (karsonkalt) [inline knowledgeBase/asset-management/asset-inventory-filters.md:70]: This is not filtering assets, but editing an asset "entity" itself.

Comment (janettelynch) [reply knowledgeBase/asset-management/asset-inventory-filters.md]: That is new to me. I thought entity is the old name and asset takes over that. I have no idea what an asset entity is.

**Review by sarabeth — APPROVED** [review]
I think that Karson did the heavy lifting here - looks good to me!

File: knowledgeBase/asset-management/asset-inventory-filters.md [hunk 3]
```diff
@@ -1,50 +1,65 @@
...
+![](../assets/assets-landing.png)
 
 Critical assets is a class of asset for which you can create queries and alerts to quickly access the most crucial data. By default, JupiterOne determines which criteria defines an asset as the most important and, therefore, the most at risk but an administrator can edit this definition.
 
 Click ![](../assets/icons/gear.png) to edit the critical asset definition default values. You can use classes, properties, and values to define what is critical.
+
 ![](../assets/asset-definition.png)
 
-Add asset classes and properties that your organization comprises a critical asset, and click **UPDATE DEFINITION**.
+Add asset classes and properties that your organization comprises a critical asset, and click **Update Definition**. Default critical asset classes include `Application`, `CodeRepo`, `Datastore`, `Function`, `Host`, and `Logs`.
```
Comment (chasen-bettinger) [inline knowledgeBase/asset-management/asset-inventory-filters.md:23]: Is `comprises` the right word here?

Comment (chasen-bettinger) [inline knowledgeBase/asset-management/asset-inventory-filters.md:23]: Is `comprises` the right word here?

**Review by chasen-bettinger — APPROVED** [review]

**Pull request merged**